LICENSE.md

```diff
@@ -1,6 +1,6 @@
 The MIT License (MIT)
 
-Copyright (c) 2015
+Copyright (c) 2016
```

Comment:
I think we want to keep this as 2015. I don't think it's legally binding, but I think technically this would be saying that only the work that was done beginning in 2016 is copyrighted.